googleTest.cc

```diff
@@ -0,0 +1,112 @@
+#include "arm64asm.h"
+#include <cstdint>
+#include <stdlib.h>
+#include <gtest/gtest.h>
+
+// // Demonstrate some basic assertions.
+// TEST(myAbs, BasicAssertions) {
+//   // Expect two strings not to be equal.
+
+//   EXPECT_EQ(myAbs(-8), 8);
+//   EXPECT_EQ(myAbs(-4294967296), 4294967296);
+//   EXPECT_EQ(myAbs(8), 8);
+//   // Expect equality.
+//   EXPECT_EQ(myAbs(0), 0);
+//   EXPECT_EQ(myAbs(4294967296), 4294967296);
+// }
+
```

Comment:
Do not need comment other TESTs, gtest supports multi tests that will show the all results in github action .